src/types/Games/Wows/Maps.ts

```diff
@@ -0,0 +1,279 @@
+import { Map } from '@/store/modules/types'
+
+export default class WowsMaps {
+  private maps: Map[] = [
+    {
+      desc: 'Somewhere in the Atlantic.',
```

Comment:
These maps will need translations if the user can see this english in the client